soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -287,6 +288,172 @@ void Randomizer::LoadHintLocations(const char* spoilerFileName) {
     }
 }
 
+std::vector<RandomizerCheck> shopItemRandomizerChecks = {
```

Comment:
I  would have preferred using the new randomizer_check_objects to retrieve this list, but the changes I'd have to make to achieve this would've ballooned up..

Reply:
#1484